modules/tinymce/CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - In some cases pressing the backspace key would incorrectly step into tables rather than remain outside #TINY-8592
 - Text alignment could not be applied to `pre` elements #TINY-7715
 - The `InsertParagraph` or `mceInsertNewLine` commands did not delete the current selection like the native command used to #TINY-8606
+- The caret moved before the selected inline boundary element when the right arrow key pressing #TINY-8601
```

Comment:
```suggestion
- The caret moved before the selected inline boundary element when the right arrow key was pressed #TINY-8601
```

or
```suggestion
- The caret moved before the selected inline boundary element when pressing the right arrow key #TINY-8601
```
or
```suggestion
- The caret moved before the selected inline boundary element when using the right arrow key #TINY-8601
```

Writing changelog items is hard. Almost as hard as naming things. :smile: not sure if my suggestions are good.

Reply:
```suggestion
- When pressing the right arrow key, the caret incorrectly moved before any selected inline boundary element #TINY-8601
```
Is another option that flips it a little?